practices/code-review.md

```diff
@@ -17,7 +17,7 @@ We manage code review via **pull requests**, which we communicate in the `#front
     * allows more junior members of staff to learn best practices quicker.
     * allows EVERYONE to learn from their colleagues.
     * facilitates people moving to other teams as they are more familiar with the codebase.
-* Ensure that Front-End Developers within each team are coding to the same guidelines.
+* Ensure that any developers, within each team, who are changing front-end templates or code are conforming to the same guidelines.
```

Comment:
the whole sentence can be deleted IMO, it's covered by "allows for standardisation of methodology across teams", above